### DISCUSSION TITLE
Building both Embededd and Non-embedded

### DISCUSSION THREAD

**Discussion by Enriquez-Johnmark** [Kyon147/laravel-shopify#302] [General]
Hi guys,
Is it possible to build both embededd and non-embedded together with this package?
I mean like some features will be embedded and the non-embedded  will be the admin portal
Any inputs will be appreciated

**Comment by Kyon147** [comment]
Non-embed is not supported currently.

If you want an admin panel, outside of the embedded app, you should be able to create new routes without shopify middleware but you would lose access to the api and would need to make your own connection or auth info.

**Reply by shahzaibfarooq** [reply]
when Non-embed version will be release ?
